apps/common-app/src/examples/RuntimeTests/ReanimatedRuntimeTestsRunner/types.ts

```diff
@@ -104,6 +104,7 @@ export type TestConfiguration = {
   render: Dispatch<SetStateAction<ReactNode | null>>;
 };
 
+/* eslint-disable no-var */
```

Comment:
This is ok, but I wonder if with newer TypeScript there's actually a nicer way to do it.